### ISSUE TITLE
It's not connecting glass textures at all. 

### ISSUE THREAD

**Issue by Seagoddess** [PepperCode1/Continuity#297]
I downloaded the mod and it's not even doing its core function. 
![2023-08-08_14 50 07](https://github.com/PepperCode1/Continuity/assets/100750139/a9116836-092e-4019-93d6-d2fd2a37d85f)
![Screenshot 2023-08-08 144958](https://github.com/PepperCode1/Continuity/assets/100750139/67aaea6e-1898-436a-947b-1d242aca7f82)
![Screenshot 2023-08-08 145033](https://github.com/PepperCode1/Continuity/assets/100750139/70a7a4f0-21ae-4d8b-84e6-4d2de91c7569)

**Comment by Seagoddess** [comment]
Turning off an on the texture pack does nothing at all

**Comment by PepperCode1** [comment]
Do you have Indium?

**Comment by Paladynee** [comment]
The mod is not working, at all. I suspected it maybe had some mod conflicts with some of the rendering optimization mods I have installed. But that was not the case. Even when running only continuity, fabric api and no resource packs, the mod simply does not work, at all.

![image](https://github.com/PepperCode1/Continuity/assets/62181440/0629ede9-0e50-486a-b44e-36a29a3d6382)
![image](https://github.com/PepperCode1/Continuity/assets/62181440/65deb29d-a7b2-4e19-8e79-e85a5a63f2af)

**Comment by PepperCode1** [comment]
Did you enable the default connected textures resource pack?

**Comment by Paladynee** [comment]
> Did you enable the default connected textures resource pack?

Wait, I didn't even realize it added resource packs. It works now... Let me try again with all the mods installed.

Update: yep, it works now. It would REALLY HELP if you modified the Modrinth page to say that the mod does not work outright, you have to enable it in the resource packs section.

**Comment by malady-L** [comment]
I've had the same issue, tried turning the texture pack on and off, nothing. Same for reloading the world/restarting the game.

**Comment by YukiAyase** [comment]
> I've had the same issue, tried turning the texture pack on and off, nothing. Same for reloading the world/restarting the game.

Please avoid necroing old threads and create your own thread
If you do create your own thread, do attach **your log file**, it can be found in **.minecraft/logs (filename is "latest")**
